defaults/main.yml

```diff
@@ -1,2 +1,4 @@
 ---
 supervisord_config_dir: "/etc/supervisor.d"
+version: 3.3.1
```

Comment:
Not sure if you really need to have it on the main playbook.  As that's the use of `/vars` and `/defaults` to pre-define values on the `role`. But yes, making it absolute in naming should be a good practice.

I'm changing it to `supervisor_version`